### PR TITLE
updated waterfox (43.0.4)

### PR DESCRIPTION
**Pull request by lamiska**

### CODE REVIEW

File: Casks/waterfox.rb [hunk 1]
```diff
@@ -1,9 +1,8 @@
 cask 'waterfox' do
-  version '40.0.2'
-  sha256 '49720ec39019cc7991531677014bbd140062fb72f0d5dbd963f7f14b75da37b6'
+  version '43.0.4'
+  sha256 'f18ecbb17af5d186ab97c85ce226c29c001cbe83d4a5d565aaf4ec435027be6a'
 
-  # windows.net is the official download host per the vendor homepage
-  url "https://waterfox.blob.core.windows.net/releases/osx64/installer/Waterfox+#{version}+Setup.dmg"
+  url "https://storage-waterfox.netdna-ssl.com/releases/osx64/installer/Waterfox%20#{version}%20Setup.dmg"
```
Comment (adidalal) [inline Casks/waterfox.rb]: Needs a comment 
`# netdna-ssl.com is the official download host per the vendor homepage`

**Comment by adidalal** [comment]
Thank you for the contribution. It needed some fixes, so they were made in de617365e68c784b1ff0985a783773e7ec958fae. Your contribution is still included (and still credited to you), with the appropriate modifications. Please feel free to ask about any of the changes.

**Comment by moontan** [comment]
Homebrew returns this:
`Error: Download failed on Cask 'waterfox' with message: Download failed: https://storage-waterfox.netdna-ssl.com/releases/osx64/installer/Waterfox%2043.0.4%20Setup.dmg`
Is the host still up to date?

**Comment by adidalal** [comment]
Yes, it is. Check for a custom `.curlrc`, that may be the problem. If you still have a problem, [open up a new issue with the appropriate template](https://github.com/caskroom/homebrew-cask#reporting-bugs).

```
[~] > brew cask audit waterfox --download
==> Downloading https://storage-waterfox.netdna-ssl.com/releases/osx64/installer
######################################################################## 100.0%
==> Verifying checksum for Cask waterfox
audit for waterfox: passed
```

**Pull request closed without merging**